src/auth/googleauth.ts

```diff
@@ -58,6 +58,8 @@ interface CredentialResult {
   default: {email: string;};
 }
 
+export interface GoogleAuthOptions extends RefreshOptions {}
```

Comment:
Same comment as above, not sure this is worth it

Reply:
Done.

Reply:
Same. I'd prefer to keep it.